src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -1603,7 +1603,13 @@ - (void)testKeepaliveWithV2API {
   GRPCTestRunWithFlakeRepeats(self, ^(GRPCTestWaiter waiterBlock, GRPCTestAssert assertBlock) {
     // The test is highly flaky when ran as part of InteropTestsRemote
     // TODO(jtattermusch): fix and re-enable the test.
-    if ([[self class] isRemoteTest]) {
+    // if ([[self class] isRemoteTest]) {
+    //   return;
+    // }
+    // This test has a race and is flaky in any configurations. See b/268379869.
```

Comment:
You can write `// TODO(b/268379869) This test ...` here, there's some tooling available around comments with that style.

Reply:
Done.